src/Microsoft.Identity.Client/Instance/AadAuthority.cs

```diff
@@ -16,6 +14,7 @@ internal class AadAuthority : Authority
         public const string DefaultTrustedHost = "login.microsoftonline.com";
         public const string AADCanonicalAuthorityTemplate = "https://{0}/{1}/";
 
+        // TODO: bogavril consolidate well known / trusted hosts
         private static readonly HashSet<string> s_trustedHostList = new HashSet<string>()
```

Comment:
>s_trustedHostList [](start = 48, length = 17)

Also this should be extensible...

Reply:
Yes, I will address this in a subsequent PR.